fundamentals/impl.tex

```diff
@@ -190,6 +191,12 @@ \subsubsection{Agent Interchangeability}\label{sec:interchangeability}
 interacting with the \Cyclus simulation kernel while abstracting away unnecessary
 detail.
 
+% I don't think this figure actually shows what it purports to show.
+% Moreover, I'm not sure that it shows anything.  Should we have a different
+% figure, or a different purpose for this figure, or entirely different text?
+% This is hinting at the archetypes paper (which I note we don't refer to
+% anywhere) and might be reimagined as a concise summary of that?
```

Comment:
You have a good point. But, maybe opening an issue is a good idea rather than these comments? I'll open a new issue to this effect and close this pr.